tests/testthat/testAnnotate.R

```diff
@@ -0,0 +1,20 @@
+# testAnnotate.R
+#
+#
+context("Gene graph annotation function")
+
+test_that("annotateGraph rejects inappropriate objects", {
+    #
+})
+
+test_that("annotateGraph has added a new \"heat\" attribute", {
+    #
+})
+
+test_that("annotateGraph annotates graph vertices for which it has no score with 0", {
+    #
+})
+
+test_that("annotateGraph properly annotates the graph", {
+    #
+})
```

Comment:
also: does not produce a broken AGG if it receives broken input - but I'm not sure right about reasonable ways in which the input could be broken. Perhaps: if an unreasonably large number of gH IDs do not match the gG IDs?